srijan_devportal.install

```diff
@@ -2,5 +2,17 @@
 
 /**
  * @file
- * Install, update and uninstall functions for the Srijan Devportal install profile.
+ * Install, update/uninstall functions for the Srijan Devportal install profile.
  */
+
+/**
+ * Implements hook_install_tasks().
+ */
+function srijan_devportal_install_tasks(&$install_state) {
+  return [
+    '\Drupal\srijan_devportal\Installer\Form\DemoInstallForm' => [
```

Comment:
We would need to change the class name here.

Reply:
This is still pending.

Reply:
done